frontend/packages/vkt/src/interfaces/clerkExamEvent.ts

```diff
@@ -10,21 +10,22 @@ interface Person {
 }
 
 export interface ClerkExamEventEnrollmentResponse
-  extends Partial<WithId>,
-    Partial<WithVersion>,
-    Omit<Enrollment, 'previousEnrollmentDate'> {
-  previousEnrollmentDate?: string;
+  extends Omit<Enrollment, 'previousEnrollmentDate'> {
+  previousEnrollmentDate: string;
 }
 
-export interface Enrollment {
-  person: Person;
+interface PartialExams {
   oralSkill: boolean;
   textualSkill: boolean;
   understandingSkill: boolean;
```

Comment:
Nämä vois siirtää Enrollment alle. Mietin myös olisko Enrollment hyvä nimetä olemaan ClerkEnrollment. Ja vielä mitä muuta, niin täällä ja publicEnrollment.ts alla hyvin samanlaisia interfaceja, pitäiskö yrittää yhtenäistää näitä? Voisi tehdä myös omana refaktorointinaan kun tää on ensin mergattu.

Reply:
Joo, noissa on samaa jonkun verran niin vois koittaa yhdistää. Päivitetään ClerkEnrollment for now